shared/actions/chat.js

```diff
@@ -72,7 +73,10 @@ const {
 
 const {conversationIDToKey, keyToConversationID, InboxStateRecord, MetaDataRecord, makeSnippet, serverMessageToMessageBody} = Constants
 
-const _selectedSelector = (state: TypedState) => state.chat.get('selectedConversation')
+const _selectedSelector = (state: TypedState) => {
+  const selected = getPath(state.routeTree.routeState, [chatTab]).last()
```

Comment:
@chrisnojima Fixed and merge conflicts resolved. @keybase/react-hackers ready for review!